### PR TITLE
Python version bump.

### PR DESCRIPTION
**Pull request by DinoBektesevic**
Updating Python to 3.10.
 Blocked by https://github.com/dirac-institute/trailblazer/pull/41 because fixes require changes to `astro_metadata_translator` installation. 
Also required explicit definition of auto-incremented id columns - which messed a bit with the dataclasses and tests.

Large cleanup of requirements - until we try it out in the wild and see if there are some additional packages that need to be pinned across the OSs.

### CODE REVIEW

**Comment by mrawls** [comment]
[Bugfixes on setup/install beyond just requirements.txt](https://trello.com/c/Tgef8OKB/32-bugfixes-on-setup-install-beyond-just-requirementstxt)

File: requirements.txt [hunk 1]
```diff
@@ -1,29 +1,8 @@
...
-sqlparse==0.4.1
-Werkzeug==1.0.1
+astropy==5.0.1
+astroquery==0.4.5
+boto3==1.21.2
+Django==4.0.2
+matplotlib==3.5.1
+moto==3.0.3
+numpy==1.22.2
+Pillow==9.0.1
```
Comment (mrawls) [inline requirements.txt:8]:
```suggestion
Pillow==9.0.1
pyqt5>=5.0
```

**Review by mrawls — COMMENTED** [review]
I _think_ this looks OK, but I'm a bit confused about how your PRs depend on each other... does this include all the commits from #41 also?

We (or at least I) really do need pyqt5 explicitly for matplotlib to work (at least on python 3.8).

Should it work with python 3.10 now, or just 3.9?

Please can haz rebase with fewer than 20,000 commits 😄

File: requirements.txt (hunk 1 above)
Comment (DinoBektesevic) [reply requirements.txt:8]: It's not quite ready yet, perhaps I should have draft PR it but then the GA tests wouldn't have run and I was testing GA, the python version was supposed to be bumped to 3.10 in GA once I was done and history cleaned up. This was just getting rid of errors and warnings I saw when locally deploying against Python 3.10. 

Sorry for the confusion.

**Comment by DinoBektesevic** [comment]
[Bump Python version to 3.10](https://trello.com/c/KkWFSQHq/35-bump-python-version-to-310)

**Comment by DinoBektesevic** [comment]
Closed because I did something to the history and I can't figure out what and is easier to fix by a new PR.

**Pull request closed without merging**